dll/cpl/sysdm/lang/fr-FR.rc

```diff
@@ -263,7 +263,7 @@ CAPTION "Variables d'environnement"
 FONT 8, "MS Shell Dlg"
 BEGIN
     SCROLLBAR IDC_DIALOG_GRIP, 245, 238, 7, 7, SBS_SIZEGRIP
-    GROUPBOX "Variables utilisateur", IDC_USER_VARIABLE_GROUP, 7, 12, 238, 100
+    GROUPBOX "User variables for ", IDC_USER_VARIABLE_GROUP, 7, 12, 238, 100
```

Comment:
```suggestion
    GROUPBOX "Variables utilisateur pour ", IDC_USER_VARIABLE_GROUP, 7, 12, 238, 100
```

Reply:
You have to open a new PR for the French translation

Reply:
You have to open a new PR for the French translation. Thanks for your approval.